game/building/package.json

```diff
@@ -4,27 +4,27 @@
   "version": "0.1.0",
   "description": "Camelot Unchained UI - building",
   "license": "MPL-2.0",
-  "main": "src/ts/main.tsx",
+  "main": "dist/building.js",
   "author": "City State Entertainment, LLC <support@citystateentertainment.com>",
   "contributors": [
-    "mehuge <mehuge@sorcerer.co.uk>",
```

Comment:
even though most of this is a complete re-write there are still a few lines of code I wrote so should not really remove me from the contributors list, but ofc your name at top though.